core/chains/evm/client/errors.go

```diff
@@ -259,6 +259,15 @@ var mantle = ClientErrors{
 	Fatal:           regexp.MustCompile(`(: |^)'*invalid sender`),
 }
 
+var hederaFatal = regexp.MustCompile(`(: |^)(execution reverted)(:|$) | ^Transaction gas limit '(\d+)' exceeds block gas limit '(\d+)' | ^Transaction gas limit provided '(\d+)' is insufficient of intrinsic gas required '(\d+)' | ^Oversized data:|status INVALID_SIGNATURE`)
+var hedera = ClientErrors{
+	NonceTooLow:           regexp.MustCompile(`Nonce too low`),
+	NonceTooHigh:          regexp.MustCompile(`Nonce too high`),
+	TerminallyUnderpriced: regexp.MustCompile(`(Gas price '(\d+)' is below configured minimum gas price '(\d+)')|(Gas price too low)`),
+	InsufficientEth:       regexp.MustCompile(`Insufficient funds for transfer| failed precheck with status INSUFFICIENT_PAYER_BALANCE`),
+	Fatal:                 hederaFatal,
+}
```

Comment:
Could you add the examples of the Hedera messages in [errors_test.go](https://github.com/smartcontractkit/chainlink/blob/c0c7649f3b4d83ad6d6cf02055f497be7dd8cf5a/core/chains/evm/client/errors_test.go#L23)? Just so we have unit test coverage for the expected messages.

Reply:
Good catch! missed merging it from the original repo to the intermediate repo, it should be available now